packages/core/src/textInput/private/Input.tsx

```diff
@@ -88,23 +90,26 @@ const Input = styled.input.attrs(({ type }) => ({
       },
     },
     inputSize === 'medium' && {
-      padding: `${rem(23)} ${rem(15)} ${rem(7)}`,
+      padding: `${isLabelPresent ? rem(23) : rem(15)} ${rem(15)} ${
```

Comment:
Can you write it more readable way like:
```
padding: ${isLabelPresent} ? `rem(23) rem(7)` : `rem(15)`
```
Are you sure about non grid specific spacing 23 & 7?

Reply:
These values are already pre existing, I did not made them.

Reply:
waiting response from design team